taxonomy.xml

```diff
@@ -381,57 +381,59 @@
        </category>
        
        <category>
-          <desc>Demons and diseases</desc>
-          <category xml:id="Barya" corresp="https://betamasaheft.eu/authority-files/Barya/main">
-             <catDesc>Bāryā</catDesc>
+          <desc>Magic and spells</desc>
+          <category>
+             <desc>Subtypes</desc>
+             <category xml:id="MagicDrawing"
+                corresp="https://betamasaheft.eu/authority-files/MagicDrawing/main">
+                <catDesc>Magic drawing </catDesc>
+             </category>
+             <category xml:id="MagicSymbols"
+                corresp="https://betamasaheft.eu/authority-files/MagicSymbols/main">
+                <catDesc>Magic symbols or letters (see also Asmat for ʾAsmāt or Secret
```

Comment:
Probably it is useful to add also Qālāt as a term, that was used by Strelcyn.

Reply:
> Probably it is useful to add also Qālāt as a term, that was used by Strelcyn.
see above
> We already have `demon `among the keywords, that can always be used together with the generic `Magic `/ `MagicText `/ `MagicFormula `keywords, and we also have `Asmat` (we can specify there that it also covers _qālāt_).